public/app/features/backup/components/BackupInventory/BackupInventory.utils.ts

```diff
@@ -0,0 +1,27 @@
+import { DataModel, Status } from './BackupInventory.types';
+import { Messages } from './BackupInventory.messages';
+
+const { status: statusMsg, dataModel: dataModelMsg } = Messages;
+
+export const formatStatus = (status: Status): string => {
+  const map: Record<Status, string> = {
+    [Status.STATUS_INVALID]: statusMsg.invalid,
+    [Status.PENDING]: statusMsg.pending,
+    [Status.IN_PROGRESS]: statusMsg.inProgress,
+    [Status.PAUSED]: statusMsg.paused,
+    [Status.SUCCESS]: statusMsg.success,
+    [Status.ERROR]: statusMsg.error,
+  };
+
+  return map[status] || '';
```

Comment:
why the `||` here? if status `null` or `undefined`, use the nullish coalescing operator instead (`??`)

Reply:
👍